aws/table_aws_codestar_notification_rule.go

```diff
@@ -0,0 +1,193 @@
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "resource",
+				Description: "The Amazon Resource Name (ARN) of the resource associated with the notification rule.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "detail_type",
+				Description: "The level of detail included in the notifications for this resource. BASIC will include only the contents of the event as it would appear in Amazon CloudWatch. FULL will include any supplemental information provided by AWS CodeStar Notifications and/or the service for the resource for which the notification is created.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "status",
+				Description: "The status of the notification rule. Valid statuses are on (sending notifications) or off (not sending notifications).",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "event_types",
+				Description: "A list of the event types associated with the notification rule.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "targets",
+				Description: "A list of targets associated with the notification rule.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "created_by",
+				Description: "The name or email alias of the person who created the notification rule.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "created_timestamp",
+				Description: "The date and time the notification rule was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "last_modified_timestamp",
+				Description: "The date and time the notification rule was most recently updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     describeNotificationRule,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     describeNotificationRule,
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("Arn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listNotificationRules(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func listCodeStarNotificationRules(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```

Please update the function naming wherever required.